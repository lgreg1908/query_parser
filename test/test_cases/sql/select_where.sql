SELECT name, age FROM People WHERE age > 30