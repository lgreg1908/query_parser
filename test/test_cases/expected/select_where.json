{
    "type": "ROOT",
    "children": [
        {"type": "DML", "value": "SELECT", "is_group": false},
        {"type": "IdentifierList", "value": "name, age", "is_group": true, "children": [
            {"type": "Identifier", "value": "name", "is_group": false},
            {"type": "Identifier", "value": "age", "is_group": false}
        ]},
        {"type": "Keyword", "value": "FROM", "is_group": false},
        {"type": "Identifier", "value": "People", "is_group": false},
        {"type": "Where", "value": "WHERE age > 30", "is_group": true, "children": [
            {"type": "Comparison", "value": "age > 30", "is_group": false}
        ]}
    ]
}